R/shared-methods.R

```diff
@@ -269,10 +269,10 @@ DataClass <- R6::R6Class(
 
       if (!is.null(self$level_data_urls[[self$level]])) {
         if (length(self$common_data_urls) > 0) {
-          self$data_urls <- unlist(merge(
+          self$data_urls <- c(
```

Comment:
doesn't this reintroduce the bug this was trying to fix when common and level have the same named entries?

Reply:
I can't see any instances where common and level have the same named entries, but if they did this could cause issues. Is there any scenario where they could have the same names?

Reply:
yes I think so and I agree it doesn't currently happen but people are silly.

Reply:
So I could either revert back to the previous way and amend the code in UK to handle this format, or if we don't really want people naming the same I could add a test for the class to check someone has not caused a naming conflict, or write a check in the code to throw an informative error about naming things the same. Happy to do either :)

Reply:
I think fix the UK I am not clear this is a problem likely to reoccur (mostly due to not understanding what is happening).